e2e/walletAvatarOptions.spec.js

```diff
@@ -8,95 +9,123 @@ const DISMISS_AVATAR_BUILDER_COORDS = { x: 20, y: 90 };
     await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
     await Helpers.checkIfExistsByText('Choose from Library');
-    await Helpers.checkIfExistsByText('Pick an Emoji');
+    if (android) {
+      await Helpers.checkIfExistsByText('Shuffle Emoji');
+    } else {
+      await Helpers.checkIfExistsByText('Pick an Emoji');
+    }
     await Helpers.checkIfExistsByText('View Profile');
-    await Helpers.tapByText('Pick an Emoji');
-    await Helpers.checkIfVisible('avatar-builder');
-    await Helpers.tapAtPoint('avatar-builder', DISMISS_AVATAR_BUILDER_COORDS);
-    await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
+    if (!android) {
+      await Helpers.tapByText('Pick an Emoji');
+      await Helpers.checkIfVisible('avatar-builder');
+      await Helpers.tapAtPoint('avatar-builder', DISMISS_AVATAR_BUILDER_COORDS);
+      await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
+    }
     await Helpers.tapByText('View Profile');
     await Helpers.checkIfVisible('profile-sheet');
     await Helpers.swipe('profile-sheet', 'down', 'slow');
   });
 
-  it('import wallet with ens but without ens avatar', async () => {
+  it('import wallet with ENS but without ENS avatar', async () => {
     await Helpers.tapAtPoint('profile-screen', WALLET_ADDRESS_COORDS);
     await Helpers.tapByText('􀂍 Add an existing wallet');
     await Helpers.typeText('import-sheet-input', process.env.TEST_SEEDS, false);
     await Helpers.waitAndTap('import-sheet-button');
     await Helpers.waitAndTap('wallet-info-submit-button');
-    await Helpers.relaunchApp();
-    await Helpers.swipe('wallet-screen', 'right', 'slow');
-    await Helpers.checkIfVisible('profile-screen');
+    await Helpers.checkIfVisible('wallet-screen', 40000);
   });
 
-  it('test imported wallet with ens but without ens avatar', async () => {
+  it('test imported wallet with ENS but without ENS avatar', async () => {
+    await Helpers.swipe('wallet-screen', 'right', 'slow');
     await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
     await Helpers.checkIfExistsByText('Choose from Library');
-    await Helpers.checkIfExistsByText('Pick an Emoji');
+    if (android) {
+      await Helpers.checkIfExistsByText('Shuffle Emoji');
+    } else {
+      await Helpers.checkIfExistsByText('Pick an Emoji');
+    }
     await Helpers.checkIfExistsByText('View Profile');
     await Helpers.checkIfExistsByText('Edit Profile');
-    await Helpers.tapByText('Pick an Emoji');
-    await Helpers.checkIfVisible('avatar-builder');
-    await Helpers.tapAtPoint('avatar-builder', DISMISS_AVATAR_BUILDER_COORDS);
-    await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
+    if (!android) {
+      await Helpers.tapByText('Pick an Emoji');
+      await Helpers.checkIfVisible('avatar-builder');
+      await Helpers.tapAtPoint('avatar-builder', DISMISS_AVATAR_BUILDER_COORDS);
+      await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
```

Comment:
Those won't work with shuffle emoji on Android. We can look for a way to test that, but unsure how.

Check text in the avatar and make sure it changes after tap?

Reply:
oh right, sounds good to me